indexd/default_settings.py

```diff
@@ -17,7 +17,7 @@
 # will be created as "<PREFIX><PREFIX><GUID>".
 CONFIG["INDEX"] = {
     "driver": SQLAlchemyIndexDriver(
-        "sqlite:///index.sq3",
+        "postgres://postgres:postgres@localhost:5432/indexd_tests",  # pragma: allowlist secret
```

Comment:
Yeah sqlite doesn't handle some of the newer data types. I moved all the test to use postgres instead of sqlite too, just to stay consistent. I saw there was a mix.